README.md

```diff
@@ -282,6 +282,17 @@ public class ExampleDomainEventHandler : IDomainEventHandler<ExampleDomainEvent>
 }
 ```
 
+## Configure the underlying Event Grid clients options
+
+You can use the [named options pattern](https://learn.microsoft.com/en-us/dotnet/core/extensions/options#named-options-support-using-iconfigurenamedoptions) to configure the behavior of the underlying Event Grid clients. For instance:
+
+```csharp
```

Comment:
Should we also support the Configure(Action) pattern? It is a common one we have in our others libraries

Something like this

```
services
    .AddPullDeliverySubscription()
    .Configure(o => o.Retry.NetworkTimeout = 49);
```

Reply:
It might not work as there's many options to configure (our own, those from event grid)